.github/workflows/tact.yml

```diff
@@ -54,17 +57,32 @@ jobs:
         run: |
           type examples\output\echo_Echo.pkg
 
+      - name: Install the package locally
+        run: |
+          npm install -g ./
+
       - name: Compare Tact version from CLI flag `--version` against package.json
         if: runner.os != 'Windows'
         run: |
-          if [ "$(./bin/tact --version)" != "$(jq -r '.version' < package.json)" ];
+          if [ "$(tact --version)" != "$(jq -r '.version' < package.json)" ];
           then false
           fi
 
+      - name: Check command line options
```

Comment:
Should I try to add such tests in this PR or it can be moved to a separate issue+PR?

P.S.: There's a maintained fork of cram, called prysk: https://github.com/prysk/prysk

Reply:
We could do something similar to the [oclif testing](https://oclif.io/docs/testing), import an async method from `bin/tact` that incapsulates the command and add test coverage. 

I wouldn't mind doing it, but I'd rather do it in a separate PR.

Reply:
I'd prefer to stay within the JS/TS/Node.js ecosystem. On the other hand, oclif testing looks like too much JS. It should be based on snapshot, i.e. tests should look a lot like shell interaction histories. Btw, the cram-like framework we are going to pick should definitely support updating those snapshots.

Reply:
Yes, let's do that testing as a separate PR